bigtable-dataflow-parent/bigtable-beam-import/README.md

```diff
@@ -114,6 +114,12 @@ Exporting HBase snapshots from Bigtable is not supported.
 
 ## Importing to Bigtable
 
+---
+**NOTE**
+
+If your snapshot or sequence files are compressed with Snappy, you'll run into errors importing them as of now. We are working on a solution to resolve this error.
```

Comment:
Snappy comressed files are not supported with Import pipelines, we are working to add support for snappy compressed files.

Reply:
Thanks, done!

Reply:
I am not sure who needs to approve for yoshi-java? Check with Igor?